src/components/Markup/markup-data.json

```diff
@@ -27,5 +27,8 @@
   "contactFormIneligible": {
     "facebookEvent": ["Contact"],
     "matomoEvent": ["Eligibilité", "Formulaire de contact inéligible - Envoi"]
+  },
+  "markdownEvent": {
+    "matomoEvent": ["Text event", "Markdown"]
```

Comment:
Text event and Markdown ? Can't you use a more precise text ?

Reply:
Sure. Could you give me the value your prefer to see appear in matomo ?